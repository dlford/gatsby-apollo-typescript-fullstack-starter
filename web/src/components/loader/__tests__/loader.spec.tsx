import React from 'react'
import { render } from '@testing-library/react'

import Loader from '../'

describe('Loader', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Loader />)
    expect(asFragment()).toMatchSnapshot()
  })
})
