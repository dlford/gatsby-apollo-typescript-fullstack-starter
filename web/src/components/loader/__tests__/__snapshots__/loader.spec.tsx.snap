// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader renders correctly 1`] = `
<DocumentFragment>
  <div
    class="css-q2e6pp"
  />
</DocumentFragment>
`;
